spec/amber/controller/base_spec.cr

```diff
@@ -33,6 +33,26 @@ module Amber::Controller
 
         TestController.new(context).render_with_layout.should eq html_output
       end
+
+      it "renders a form with a csrf tag" do
+        request = HTTP::Request.new("GET", "/?test=test")
+        context = create_context(request)
+        html_output = <<-HTML
```

Comment:
you can move this into a seed file and read the file contents to make the spec more cleaner.